src/main/java/com/couchbase/client/dcp/config/ClientEnvironment.java

```diff
@@ -56,6 +57,11 @@
     private final String password;
 
     /**
+     * Time in milliseconds to wait before retrying bootstrap.
+     */
+    private final long bootstrapTimeout;
```

Comment:
with "bootstrap" I meant that this is timeout not for whole connect action, but only for part of it, which related to waiting for initial config